src/test/java/org/openmrs/contrib/qaframework/automation/VitalsAndTriagingSteps.java

```diff
@@ -135,6 +138,41 @@ public void savePatientVitals() {
 
 	@Then("the system adds patient vitals into the vitals table")
 	public void systemAddsPatientVitals() {
-		assertTrue(patientCaptureVitalsPage.containsText("Vitals"));
+		assertNotNull(visitsDashboardPage.getEncountersCount());
```

Comment:
`getEncountersCount()` is too generic. We would rather assert what is being performed by step `the system adds patient vitals into the vitals table`.

Here are two options that might be worth;

1. use the current assertion present in the method "systemAddsPatientVitals()"
2. asserting that the vitals table is not null after saving the vitals. This would mean adding a method that returns the vitals list from the vitals table within the PatientVisitsDashbaordPage . 
```js
public List<WebElement> getVitalsList() {
       return findElements(VITALS_LIST);
}
```

then invoke the method within the selenium test 

```js
@Then("the system adds patient vitals into the vitals table")
public void systemAddsPatientVitals() {
       assertNotNull(visitsDashboardPage.getVitalsList());
}
```

Reply:
@kdaud going with the current assertion present in the method systemAddsPatientVitals() may not be a long lasting solution since it is based on visible text. I  am imagining if the application is being used in a different language. 

Let me get a work around the second option.